index.bs

```diff
@@ -1230,6 +1086,175 @@ spec:css-syntax-3;
+
+    :   <dfn>optional</dfn>
+    ::  If credentials can be handed over for a given operation without user mediation, they will
+        be. If [=requires user mediation|user mediation is required=], then the user agent will
+        involve the user in the decision.
+
+        Note: This is the default behavior for {{get()}} and {{create()}}, and is intended to serve a case where a
+        developer has reasonable confidence that a user expects to start an operation. If
+        a user has just clicked "sign-in" for example, then they won't be surprised or confused to
+        see a [=credential chooser=] if necessary.
+
+    :   <dfn>conditional</dfn>
+    ::  For {CredentialsContainer/get()}, discovered credentials are presented to the user in a non-modal dialog along with an
+        indication of the [=origin=] which is requesting credentials. If the user makes a gesture
+        outside of the dialog, the dialog closes without resolving or rejecting the {{Promise}}
+        returned by the {{CredentialsContainer/get()}} method and without causing a user-visible
+        error condition. If the user makes a gesture that selects a credential, that credential is
+        returned to the caller. The [=prevent silent access flag=] is treated as being `true`
+        regardless of its actual value: the {{CredentialMediationRequirement/conditional}} behavior
+        always involves [=user mediation=] of some sort if applicable credentials are discovered.
+
+        If no credentials are discovered, the user agent MAY prompt the user to take action in a way
+        that depends on the type of credential (e.g. to insert a device containing credentials).
+        Either way, the `get()` method MUST NOT resolve immediately with `null` to avoid revealing
+        the lack of applicable credentials to the website.
+
+        Websites can only pass {{CredentialMediationRequirement/conditional}} into the
+        {{CredentialsContainer/get()}} method if all of the [=relevant credential interface objects|credential
+        interfaces it refers to=] have overridden {{Credential/isConditionalMediationAvailable()}} to return
+        a new {{Promise}} that [=resolves=] with `true`.
+
+        For {CredentialsContainer/create()}, if a user has previously consented to credential creation and
+        the user agent knows it recently mediated an authentication, then the `create()` call may resolve without
+        additional prominent modal interaction. If the user agent did not recently mediate an authentication or
+        does not have consent for credential creation, then the call must throw a "{{NotAllowedError}}" {{DOMException}}.
+
+        Websites can only pass {{CredentialMediationRequirement/conditional}} into the
+        {{CredentialsContainer/create()}} method if |conditionalCreate| is present in the result of
+        {{PublicKeyCredential.getClientCapabilities()}} defined in [[WEBAUTHN]].
```

Comment:
This paragraph implies that conditional creation is supported for all credential types if |conditionalCreate| is present in `getClientCapabilities()`. Let's make clear it's only available for webauthn credentials:

```suggestion
        Websites can only pass {{CredentialMediationRequirement/conditional}} into the
        {{CredentialsContainer/create()}} method for {{PublicKeyCredential}} requests, and
        only if {{ClientCapability/conditionalCreate}} is present in the result of
        {{PublicKeyCredential.getClientCapabilities()}} as defined in [[WEBAUTHN]].
```

I'm not too happy that this webauthn-specific detail is leaking into credman, but it's probably okay to live with it for now. If in the future more credential types wanted to support this, we could make a more generic interface to feature detect.